tests/Database/DatabaseEloquentIntegrationTest.php

```diff
@@ -1202,6 +1202,7 @@ public function testIsAfterRetrievingTheSameModel()
     public function testFreshMethodOnModel()
     {
         $now = Carbon::now();
+        $nowSerialized = $now->startOfSecond()->toJSON();
```

Comment:
The reason why I used `startOfSecond` here is because the date is being stored in the DB without the fractions (which get rounded down).

Reply:
Added a test.